src/components/useFetch.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { useState, useEffect } from 'react'
```

Comment:
Could you please add `onSuccess, onError & refetch` functions as hook props please?

Reply:
Added :-)